test/e2e/storage/persistent_volumes.go

```diff
@@ -313,6 +313,32 @@ var _ = utils.SIGDescribe("PersistentVolumes", func() {
 				pvc = nil
 			})
 		})
+
+		ginkgo.Context("delete pv prior", func() {
+			ginkgo.BeforeEach(func() {
+				ginkgo.AfterEach(func() {
+					framework.Logf("AfterEach: Cleaning up test resources.")
+					if errs := e2epv.PVPVCCleanup(c, ns, pv, pvc); len(errs) > 0 {
+						framework.Failf("AfterEach: Failed to delete PVC and/or PV. Errors: %v", utilerrors.NewAggregate(errs))
+					}
+				})
+			})
+
+			// Waiting for the pvc and pvc bounded and then delete the pv prior
+			ginkgo.It("to exercise deletion of PV prior to deletion of PVC for a Bound PV-PVC pair", func() {
+				pv, pvc, err = e2epv.CreatePVPVC(c, f.Timeouts, pvConfig, pvcConfig, ns, false)
+				framework.ExpectNoError(err)
+				// 1. verify that the PV and PVC have bound correctly
+				ginkgo.By("Validating the PV-PVC binding")
+				framework.ExpectNoError(e2epv.WaitOnPVandPVC(c, f.Timeouts, ns, pv, pvc))
+
+				// 2. delete PV prior to deletion of PVC for a Bound PV-PVC pair
+				ginkgo.By("Wating for the pv deleted")
```

Comment:
Where is the call to delete the PV API object before deleting PVC API object?  I don't see it.

Reply:
WaitForPersistentVolumeDeleted